bsp/esp32_p4_function_ev_board/esp32_p4_function_ev_board.c

```diff
@@ -393,7 +451,11 @@ lv_display_t *bsp_display_start(void)
         .buffer_size = BSP_LCD_DRAW_BUFF_SIZE,
         .double_buffer = BSP_LCD_DRAW_BUFF_DOUBLE,
         .flags = {
+#if CONFIG_BSP_LCD_COLOR_FORMAT_RGB888
+            .buff_dma = false,
+#else
             .buff_dma = true,
+#endif
             .buff_spiram = false,
```

Comment:
```suggestion
            .buff_dma = false,
            .buff_spiram = true,
```
I tested that using PSARM directly here would also provide good performance. And we can increase `BSP_LCD_DRAW_BUFF_SIZE` to `BSP_LCD_H_RES * BSP_LCD_V_RES`. You can try it with the [esp-ui example](https://gitlab.espressif.cn:6688/ae_group/esp-ui/-/tree/feat/add_example_p4_func_ev_board/examples/esp_idf/esp32_p4_function_ev_board?ref_type=heads).

Reply:
I tried benchmark demo:
1. Internal buffer, RGB565, DMA (BSP_LCD_H_RES * 100): 24 FPS (avg)
2. SPIRAM buffer, RGB565 (BSP_LCD_H_RES * 100): 18 FPS (avg)
3. SPIRAM buffer, RGB565 (BSP_LCD_H_RES * BSP_LCD_V_RES): 19 FPS (avg)

I think, we should keep the best performance settings. What do you think @tore-espressif ?

Reply:
I think that making the best performance default is a good idea. If someone needs to use different configuration, we provide `bsp_display_start_with_config()` function